### PR TITLE
Fixed graph

### PR DESCRIPTION
**Pull request by joewilliams007**
Actually implemented the graph to work with the main loop.

Added customization options for the graph.

For testing purposes: Please set boolean testGraphs = true; in SimulationController line 69.

### CODE REVIEW

File: src/com/biocollapse/controller/SimulationController.java [hunk 1]
```diff
@@ -62,9 +63,15 @@ public void runMainLoop() {
 
                 double fps = lastFps; // Needed for swingutilities to access scope.
                 SwingUtilities.invokeLater(() -> {
-                    // TODO: Get actual statistics.
                     LiveStatistics newLiveStatistics = simulationService.calculateLiveStatistics(humans, hospitals);
-                    visualisation.update(humans, newLiveStatistics, fps);
+
+                    // For testing, please set testGraphs to true. TODO: Remove testing code later.
+                    boolean testGraphs = false;
+                    if (testGraphs) {
```
Comment (N3moAhead) [inline src/com/biocollapse/controller/SimulationController.java]: Please remove the test code before merging the pr. 😉

**Review by N3moAhead — APPROVED** [review]
The Code looks very good to me! And it also works without problems 👍🏻  Well Done! 👍🏻 

It would be great if you could remove the testing code before merging and then we are ready to go 👍🏻

File: src/com/biocollapse/controller/SimulationController.java (hunk 1 above)
Comment (joewilliams007) [reply src/com/biocollapse/controller/SimulationController.java]: Done! Thanks for feedback. Glad it works now.

**Comment by joewilliams007** [comment]
Thanks for the feedback!

**Pull request merged**